src/api/app/views/webui2/webui/project/_pulse_list_builds.html.haml

```diff
@@ -0,0 +1,15 @@
+- if @builds.any?
+  - builds = @builds.group(:event_type).count
+  There have been
```

Comment:
There **has** been

Reply:
Still valid

Reply:
As discussed with @henne, it depends on the number of builds. So it can be has or have (so we use pluralize)